> @wangnuannuan could you please explain which problem are you trying to solve?
> I.e. maybe an example for mere mortals, as to me at least it's not clear what's wrong with current implementation.
> I mean your fix might be entirely correct, but first I'd like to clearly understand what's no OK w/o it.


Our CI job `arcgnu_verif_docs_website` failed :
```
make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v4.0.2

Configuration error:
There is a syntax error in your configuration file: (unicode error) 'unicodeescape' codec can't decode bytes in position 0-1: truncated \uXXXX escape (conf.py, line 224)

make: *** [html] Error 2
make pdf

sphinx-build -b pdf -d _build/doctrees   . _build/pdf

Running Sphinx v4.0.2

Configuration error:

There is a syntax error in your configuration file: (unicode error) 'unicodeescape' codec can't decode bytes in position 0-1: truncated \uXXXX escape (conf.py, line 224)
 
 make: *** [pdf] Error 2
```

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -221,7 +221,7 @@ latex_elements = {
 
 # Additional stuff for the LaTeX preamble.
 # Increase limitations on the number of floats.
-'preamble': '\usepackage{float}',
+'preamble': r'\usepackage{float}',
 
 # Latex figure (float) alignment to fix image location in the document.
 'figure_align': 'H',

